cpp/include/cudf_test/base_fixture.hpp

```diff
@@ -144,6 +144,13 @@ class UniformRandomGenerator {
   {
   }
 
+  /**
+   * @brief Construct a new Uniform Random Generator to generate uniformly random booleans
+   *
```

Comment:
`tparam` for `TL`

Reply:
`tparam` should be ideally be documented, but since it's default assigned, doxygen does not complain. (that's also another reason why doxygen does not complain about SFINAE tparam too).